src/main/java/seedu/address/model/util/SampleDataUtil.java

```diff
@@ -23,38 +23,38 @@
         return new StudySpot[] {
             new StudySpot(new Name("Starbucks UTown"),
                     new Rating("4"),
-                    new Email("-"),
+                    new OperatingHours("0900-2200, 0900-2200"),
                     new Address("University Town, ERC, 138608"),
                     new Favourite(true),
                     getTagSet("coffee", "tea", "smellsGood"),
                     getAmenitySet("wifi", "charger", "food", "aircon")),
             new StudySpot(new Name("COM1 Tech Hangout"),
                     new Rating("4"),
-                    new Email("-"),
+                    new OperatingHours("unspecified"),
```

Comment:
Should we use '-' or 'unspecified'? I personally think '-' looks nicer, but would want to know what the others think @ruoyann @joenzkimchan @mslevis